R/usage_to_annual_nets_distributed.R

```diff
@@ -0,0 +1,234 @@
+#' Convert bednet usage to annual nets distributed per capita
+#' 
+#' Assumptions: 
+#' - Net use rates stays at 2019 levels for all countries
+#' - When converting from access to NPC, use the 2020 loess curve fit of access-NPC 
+#'   at the country-month level.
+#' 
+#' @param target_usage A single value or vector of desired target usages to model.
+#' @param country_iso3 Vector of ISO3 codes of African countries for which to return results. 
+#' Default = "all".
+#' @param extrapolate_npc Option to extrapolate target nets per capita beyond fitted 
+#' Loess curve. Default = "linear" for linear extrapolation. For any other inputs,
+#' NA outputs are returned for any target usages exceeding currently observed levels.
+#' @param k Fixed rate parameter of the internal function of net loss, used to 
+#' estimate the median net retention times by country. Default = 20 from
+#' Bertozzi-Villa, Amelia, et al. Nature communications 12.1 (2021): 1-12.
+#'
+#' @return Dataframe of access, nets per capita and annual nets distributed per capita
+#' for the desired steady-state target usage by country
+#' 
+#' @importFrom stats 'approx' 'reshape' 'loess' 'predict'
+#' @export
+
+find_annual_nets_distributed <- function(target_usage, country_iso3="all", 
+                                         extrapolate_npc = "linear", k=20) {
```

Comment:
This seems a bit pedantic (sorry!) but good to follow a consistent style guide (e.g. spaces arounf <- and =).
The tidyverse style guide is a good reference here: https://style.tidyverse.org/index.html